docs/config.toml

```diff
@@ -28,6 +28,11 @@ menushortcutsnewtab = true # set true to open shortcuts links to a new tab/windo
 enableGitInfo = true
 operatorVersion = "5.0.3"
 centosBase = "centos8"
+imageCrunchyPostgresCentos = "registry.developers.crunchydata.com/crunchydata/crunchy-postgres:centos8-13.4-1"
```

Comment:
Do we need to have centos in the param here? Could we just use `imageCrunchyPostgres` :question: same for the UBI tag

Reply:
I just did that in case for some reason we needed to delineate between the two.  I'm fine with removing it if we will only ever have centos for those images and ubi for the operator tag